config/readme.md

```diff
@@ -0,0 +1,45 @@
+# Environment configuration
+
+Config settings for the environment go in this directory. Mothership will check the environment set by the `COG_ENV`
+server variable. It will first check the `config/[environment name]` directory, and if it cannot find the config
+setting, it will check the `config` directory. For instance, when setting the database, your base `config/db.yml` config file
+might look like this:
+
+    hostname: localhost
+    user: root
+    pass: password
+    name: mothership_db
+    charset: utf8
+    cache: mysql_memory
+
+However, in the live environment, it will be on a server and so the database user, password and schema will probably
+be different, so your `config/live/db.yml` file will might look like this:
```

Comment:
@kuiche because it makes it more readable in my editor, i.e. I don't need to scroll right to see what the line says. It doesn't impact the markdown as you need a blank line to do a linebreak in md